themes/sensei-course-theme/templates/lesson.html

```diff
@@ -46,11 +46,14 @@
 
     <!-- wp:column {"width":"","className":"sensei-course-theme__main-content"} -->
     <div class="wp-block-column sensei-course-theme__main-content">
-        <!-- wp:sensei-lms/course-theme-lesson-module /-->
-        <!-- wp:post-title {"level": 1} /-->
-        <!-- wp:sensei-lms/course-theme-notices /-->
-        <!-- wp:post-content /-->
-        <!-- wp:sensei-lms/quiz-actions /-->
+        <!-- wp:group {"className":"sensei-course-theme__main-content__container", "tagName": "main"} -->
+        <main class="wp-block-group sensei-course-theme__main-content__container">
```

Comment:
Ah, I just noticed one other thing. It introduced a class `wp-container-61fc29d7ee8ee` with a margin-top and bottom as `0` to all children elements. So it removed the margins from the module name and lesson title.

It's not a big change, so I don't know if we should really worry about that. Maybe it's worth checking the purpose of this class, and if we keep that, we could remove the margins we have in that elements styles.

Reply:
That's pretty bad :( I disabled it via a filter in [1371672](https://github.com/Automattic/sensei/pull/4749/commits/13716720e5154e848c9886595138da076313c817)